### DISCUSSION TITLE
[Art competition] Cryptography Cracking

### DISCUSSION THREAD

**Discussion by Re-st** [prosyslab-classroom/cs348-information-security#411] [Art competition]
![Astronomer searching for a very distant star in the sky in the deep night](https://github.com/prosyslab-classroom/cs348-information-security/assets/42648067/81806481-89f5-46b2-a809-5cb3e16a22dc)

## Short Description
For me, [Cryptography Cracking] means [Astronomer searching for a very distant star in the sky in the deep night]".

## Long Description (up to 3 sentences)
You are an astronomer searching for a star in a very specific condition. The whole celestial sphere is open to everyone, but only when you know the correct and precise spot to search then you can find the star in limited time. Finding a vulnerability of Encryption Scheme (like the padding oracle for CBC) is important and central in cracking the code, too.

**Comment by KihongHeo** [comment]
Hope you find your shining star.
